canonical-websites/partners.ubuntu.com
# Remove old styling and add Vanilla Framework 1.8. NOTE: all pages will have broken styling.

## Done

Added Vanilla Framework v1.8 as a requirement and removed the old styling

## QA

./run and see that the site still builds and runs. The styling on all pages will be absent.
